### ISSUE TITLE
Provide mechanism to unpack ipk in crossenv

### ISSUE THREAD

**Issue by virtuald** [robotpy/robotpy-cross-docker#2]
Maybe a script or something?

There are two types of ipks, one is things we converted from python wheels, and others are native packages. I'd like to be able to unpack both types? 

A good testcase for this is opencv and robotpy-cscore.

@DirtyJerz any chance you can take a look at this?

**Comment by virtuald** [comment]
Ok, so we have the right content in the crossenv -- but the next question then is where do we actually extract to? I'm not 100% sure where the crossenv compiler is going to be looking for installed things, but maybe we can provide an override?

**Comment by virtuald** [comment]
> if the site-packages works, I can build the opkg manager from source in the image then it's just $ opkg install. If one of the venvs is needed, I can make a script in /usr/local/bin called "opkg-venv" which would unpack the ipk into the currently activated venv.
if this is needed to install reqs then i'm guessing it needs to be installed into the venv.

The goal would be to install the prereqs in the crossenv from ipk, and be able to link against those libraries. A good testcase is building robotpy-cscore, which I imagine would be something like this:

* Download opencv-dev.ipk
* Install opencv-dev.ipk in env
* Run `python setup.py bdist_wheel` against robotpy-cscore repo, and it correctly finds the OpenCV headers and links against OpenCV

Does that make more sense?

**Comment by DirtyJerz** [comment]
Yes. That should be doable. Just need to unzip the data.tar.gz to venv. I'll whip up a bash script tomorrow that will do this.
